src/controllers/expenses.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const userServices = require('../services/users');
+const expenseServices = require('../services/expenses');
+
+const getAllExpenses = (req, res) => {
+  const filteredExpenses = expenseServices.filterExpenses(req.query);
+
+  res.send(filteredExpenses);
+};
+
+const getExpense = (req, res) => {
+  const { expenseId } = req.params;
+  const foundExpense = expenseServices.getExpenseById(expenseId);
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundExpense);
+};
+
+const addExpense = (req, res) => {
+  const {
+    userId,
+    spentAt,
+    title,
+  } = req.body;
+
+  const users = userServices.getAllUsers();
+  const usersIds = users.map(user => user.id);
+  const isValidUser = usersIds.includes(userId);
+
+  if (!isValidUser || !spentAt || !title) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const expense = expenseServices.createExpense(req.body);
+
+  res.statusCode = 201;
```

Comment:
you can use the method for it
```suggestion
  res.status(201);
```